### PR TITLE
Create single.html template for documentation

### PR DESCRIPTION
**Pull request by celestehorgan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+3. Install Netlify with access to your documentation site repository.
+4. Leave all other settings as default and click **Deploy Site**.
+
+# What's included
+
+This repository has two layouts with minimal styling, all stored under `/layouts/_default`:
+
+A **homepage** template, a basic homepage which uses the following:
+  * The `baseof.html` file and partials in the `/partials/home` directory
```
[the remark this answers is not in the document]
Comment (celestehorgan) [reply README.md]: Good catch :+1:

File: assets/sass/_docs.sass [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
+.title
+  @extend .title
+  @extend .is-1
+  font-weight: 200
+  margin-top: 1rem
+  margin-bottom: 1.5rem
+
+#navsection .menu-label
```
[the remark this answers is not in the document]
Comment (celestehorgan) [reply assets/sass/_docs.sass]: This was actually a remnant of something which I eventually removed – I'll remove the entire class for now.

File: assets/sass/_docs.sass [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+.hashlink
+  @extend .icon
+  font-size: 1rem
+  height: 1rem
+  width: 1rem
+
+.headline-hash
+  display: none
+
+h1:hover > .headline-hash, h2:hover > .headline-hash, h3:hover > .headline-hash,
```
Comment (lucperkins) [inline assets/sass/_docs.sass]: Bonus points if you can clean this up with a [Sass @for loop](https://sass-lang.com/documentation/at-rules/control/for).

File: assets/sass/_docs.sass [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+.hashlink
+  @extend .icon
+  font-size: 1rem
+  height: 1rem
+  width: 1rem
+
+.headline-hash
+  display: none
+
+@for $i from 1 through 6
```
Comment (lucperkins) [inline assets/sass/_docs.sass]: You get 💯 cool points, as promised

**Pull request merged**